<launch>
    <arg name="wrench_topic" default="/leptrino/force_torque/filtered" />
    <arg name="wrench_statistics_service" default="/wrench_statistics" />
    
    <node name="wrench_statistics" pkg="grinding_force_torque" type="wrench_statistics.py" output="screen">
        <param name="wrench_topic" value="$(arg wrench_topic)" />
        <param name="wrench_statistics_service" value="$(arg wrench_statistics_service)" />    
    </node>

</launch>